metals/src/main/scala/scala/meta/internal/metals/ImportedBuild.scala

```diff
@@ -3,8 +3,11 @@ package scala.meta.internal.metals
 import java.{util => ju}
 
 import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
 
 import ch.epfl.scala.bsp4j._
+import ch.epfl.scala.{bsp4j => b}
```

Comment:
We already import `import ch.epfl.scala.bsp4j._`, so this should not be needed.

Reply:
Ah duh, fixed.